Origin: upstream, https://github.com/cyrusimap/cyrus-sasl/commit/ee592d7be6338bacac69657897deed3a715f199e
From: Bastian Germann <bage@debian.org>
Date: Mon, 21 Jan 2019 15:53:54 +0000
Subject: Stop importing docutils_version in sphinx-build manpage helper

The import is unused and will fail with newer sphinx versions.

Reported-by: Ondřej Surý <ondrej@sury.org>
Signed-off-by: Bastian Germann <bage@debian.org>
---
 docsrc/exts/sphinxlocal/writers/manpage.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docsrc/exts/sphinxlocal/writers/manpage.py b/docsrc/exts/sphinxlocal/writers/manpage.py
index 4bdde126..c33ba617 100644
--- a/docsrc/exts/sphinxlocal/writers/manpage.py
+++ b/docsrc/exts/sphinxlocal/writers/manpage.py
@@ -23,7 +23,6 @@
 
 from sphinx import addnodes
 from sphinx.locale import admonitionlabels, _
-from sphinx.util.compat import docutils_version
 
 class CyrusManualPageWriter(ManualPageWriter):
 
